code/software/libs/src/libm/common/Makefile.am

```diff
@@ -0,0 +1,76 @@
+## Process this file with automake to generate Makefile.in
```

Comment:
Does this still work? If not, I'd prefer we omit the automake files and just commit the generated Makefile...